pkg/bmc/ilo5.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2016-2018 Hewlett Packard Enterprise Development LP
+
+package bmc
+
+import (
+	"net/url"
+)
+
+func init() {
+	registerFactory("ilo5", newILO5AccessDetails, []string{})
```

Comment:
Same here with https.

Reply:
@zaneb Thanks for the comment. Made the change.